libs/server/repository/api-rest/src/lib/controllers/timeline.controller.ts

```diff
@@ -0,0 +1,42 @@
+import { ApiBearerAuth, ApiOkResponse, ApiTags } from '@nestjs/swagger';
+import { Controller, Get, Param, Query, UseGuards, UsePipes, ValidationPipe } from '@nestjs/common';
+import { AuthGuard, Credentials, RequestCredentials } from '@pimp-my-pr/server/auth/public';
+import {
+  GetPrTimelineQuery,
+  TimelineFacade
+} from '@pimp-my-pr/server/repository/core/application-services';
+import { PrTimelineReadModel } from '@pimp-my-pr/server/repository/core/application-services';
+import { GenerateTimelineDto } from '../dtos/generate-timeline.dto';
+import { decreaseDateByTimelineStep } from '@pimp-my-pr/server/repository/core/domain';
+
+@ApiTags('timeline')
+@UseGuards(AuthGuard)
+@ApiBearerAuth()
+@Controller('timeline')
+export class TimelineController {
+  constructor(private timelineFacade: TimelineFacade) {}
+
+  @ApiOkResponse({ type: [PrTimelineReadModel] })
+  @UsePipes(
+    new ValidationPipe({ transform: true, transformOptions: { enableImplicitConversion: true } })
```

Comment:
Isn't it set in the global scope? Was that necessary?

Reply:
in the global scope the ValidationPipe has no {transform: true} parameter which is required to use @Transform decorator in the dto

Reply:
maybe in the global scope also should be used transform: true ?